### PR TITLE
Show path / parent folder to distinguish files

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/issues/5953

## Screenshots
favorites
search

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

## Open tasks:

- [ ] ...

### CODE REVIEW

File: src/components/organisms/OcResource/OcResource.vue [hunk 1]
```diff
@@ -43,8 +43,21 @@
         :full-path="resource.path"
         :is-path-displayed="isPathDisplayed"
       />
-      <div v-if="resource.indicators.length > 0" class="oc-resource-indicators">
-        <oc-status-indicators :resource="resource" :indicators="resource.indicators" />
+      <div class="oc-resource-indicators">
+        <router-link
```
Comment (fschade) [inline src/components/organisms/OcResource/OcResource.vue:47]: is the resource path guaranteed to be always a link? What if we just want to use it to transport some meta information without having it routable?

File: src/components/organisms/OcResource/OcResource.vue [hunk 2]
```diff
@@ -43,8 +43,21 @@
...
+          v-if="isPathDisplayed"
+          :to="parentFolderLinkPath"
+          :style="parentFolderStyle"
+          class="parent-folder"
+        >
+          <oc-icon name="folder-2" size="small" fill-type="line" />
+          <span class="text" v-text="parentFolder" />
+        </router-link>
+        <oc-status-indicators
+          v-if="resource.indicators.length > 0"
```
Comment (fschade) [inline src/components/organisms/OcResource/OcResource.vue:57]: can be simplified `resource.indicators.length`

File: src/components/organisms/OcResource/OcResource.vue [hunk 3]
```diff
@@ -135,8 +149,35 @@ export default {
...
+
+    parentFolderStyle() {
+      const hasLinkTarget = this.targetRoute !== null
+      return {
+        cursor: hasLinkTarget ? "pointer" : "default",
+      }
+    },
+
+    parentFolderLinkPath() {
+      console.log(this.resource.path)
```
Comment (fschade) [inline src/components/organisms/OcResource/OcResource.vue:167]: dev leftover

**Review by fschade — COMMENTED** [review]
found some smaller issues

File: src/components/organisms/OcResource/OcResource.vue [hunk 4]
```diff
@@ -135,8 +150,38 @@ export default {
...
   },
-
   computed: {
+    parentFolderComponentType() {
+      return this.targetRoute !== null ? "router-link" : "span"
+    },
+
+    parentFolder() {
+      const folder = path.basename(path.dirname(this.resource.path)).replace(".", "")
+      if (folder !== "") return folder
```
Comment (fschade) [inline src/components/organisms/OcResource/OcResource.vue:160]: `return folder !== "" ? folder : this.$gettext("All Files and Folders")`

**Review by fschade — APPROVED** [review]
one optional finding 👍

File: src/components/organisms/OcResource/OcResource.vue [hunk 5]
```diff
@@ -251,6 +296,29 @@ export default {
 
   &-indicators {
     display: flex;
+
+    .parent-folder {
+      display: flex;
+      align-items: center;
+      margin-right: 10px;
```
Comment (kulmann) [inline src/components/organisms/OcResource/OcResource.vue:303]: line can be removed

File: src/components/organisms/OcResource/OcResource.vue [hunk 6]
```diff
@@ -251,6 +296,29 @@ export default {
...
+
+      padding: 0 2px 0 2px;
+      margin: 0 8px 0 -2px;
+
+      .oc-icon {
+        padding-right: 3px;
+      }
+
+      .text {
+        font-size: 13px;
```
Comment (kulmann) [inline src/components/organisms/OcResource/OcResource.vue:313]: please use a `rem` value

**Review by kulmann — APPROVED** [review]

**Pull request merged**